### ISSUE TITLE
Would  you please share the checkpoint and its corresponding hyperparameters with me?

### ISSUE THREAD

**Issue by mengfeidu** [cwhao98/DDL#2]
Hello, thank you for your excellent work. I am currently attempting to implement your model in my local environment. However, I am facing difficulties in achieving the performance of the R2R test dataset displayed in your paper. While the performance in Val Unseen is similar to what you reported, it is notably lower in the test dataset. I was wondering if it would be possible for you to share the checkpoint of your best model and its corresponding hyperparameters with me. Thank you in advance!

**Comment by cwhao98** [comment]
Hi,

Thanks for your interest. Previously we conducted the experiments in a foreign server which is not available now. Sorry for the inconvenience. But we have two suggestions for your issue.
    

- The performance maybe affected by different local environments. We use a language auxiliary loss to regularize the attention weight. Empirically, we find the loss weight (--LWeight and --AWeight) will make the performance a bit change when ranging from 0.5 to 2.0.

- The agent takes a long time to converge, and sufficient training steps may be helpful.  The machine we used has the limitation of two days in one trial, so we often train the model in a "resume" manner.

Thanks again.
